src/Qwiq.Core.Rest/Qwiq.Client.Rest.csproj

```diff
@@ -1,125 +1,29 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\..\build\targets\common.props" />
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
+
   <PropertyGroup>
-    <ProjectGuid>{0F70E1C2-D696-4749-8601-374A7C9C268A}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <!-- REST client targets net8.0 only (cross-platform) -->
```

Comment:
The underlying TFS SDK (v15) only supports net472 (maybe even net462) and netstandard2.0 (possibly, need to research this). How can we move up to net8.0 without incurring a slew of warnings from NuGet indicating the target is different and there may be a compatibility issue?